tests/unit/test_checkpointing.py

```diff
@@ -152,8 +152,8 @@ def checkpoint_correctness_verification(args,
         compare_lr_scheduler_states(trained_model, loaded_model)
 
 
-@pytest.mark.skipif(not deepspeed.ops.__installed_ops__['lamb'],
-                    reason="lamb is not installed")
+#@pytest.mark.skipif(not deepspeed.ops.__installed_ops__['lamb'],
```

Comment:
Should these now be deleted? Or switch to a compatibility check? Same comment for other unit test changes.